src/utils/schema/resolvers/Entity/index.ts

```diff
@@ -61,15 +61,17 @@ export const getEntityResolver = (name: string, data, id: string, ids) => {
       Object.assign({}, resolvers, {
         [fieldName]: (entity, args, context) => {
           const field = fields[fieldName];
-          const value = relationshipFields.includes(fieldName) ?
+          let value = relationshipFields.includes(fieldName) ?
             entity.data[fieldName.substr(0, fieldName.length - (fieldName.endsWith('_id') ? 3 : 4))] :
             entity.data[fieldName];
-          if (context.display && (args.display === undefined || args.display)) {
-            const formField = data[name].find(x => x.name === fieldName);
-            if (formField && (formField.choices || formField.choicesByUrl)) {
+          const formField = data[name].find(x => x.name === fieldName);
+          if (formField && (formField.choices || formField.choicesByUrl)) {
+            value = [...new Set(value)];
```

Comment:
@matheus-relief be aware that the [...set( ) ] only works if the question is multi choices, so that breaks all dropdown / radiogroup questions:
![Capture d’écran 2022-06-02 à 11 30 46](https://user-images.githubusercontent.com/50317271/171600962-fa0fd64c-8df4-492b-8c89-4f1a7029c9b9.png)